src/components/managersinglecomm/managersinglecomm.module.css

```diff
@@ -0,0 +1,45 @@
+/* ============= Card CSS =============== */
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  width: 30%;
+  border-radius: 5px;
+  padding-bottom: 20px;
+}
+
+/* ============= Card Image =============== */
+
+.img {
+  border-radius: 5px 5px 0 0;
+  width: 100%;
+}
+
+/* ============= Card Body =============== */
+.cardBody {
+  padding: 2px 16px;
+  text-align: center;
+  margin: 2rem 0 2rem 0;
```

Comment:
> Not sure why we are mixing `rem` and `px` units? Rationale?

Certain units help control spacing a bit better for me. I've looked at other CSS modules in the repo, some are using mixed units, so I didn't think it would be an issue to do so. I can ask the group which units we'd like to stick to when starting to design because I've seen 'rem' 'em' and 'px'.